### DISCUSSION TITLE
How to add table extension to menu?

### DISCUSSION THREAD

**Discussion by ghost** [remirror/remirror#783] [Q&A] [answered]
```
 <button
    onClick={() => commands.toggleTable()}
     style={{ fontWeight: active.underline() ? 'bold' : undefined }}
  >
  table
 </button>
```
This is not working.

Can you provide general steps to include any extension in menu?

**Comment by ifiokjr** [comment, marked as answer]
This is better as a discussion topic. 

You need to add the `TableExtension` and then call `commands.createTable()`.
